lambdas/report-user-identity/src/main/java/uk/gov/di/ipv/core/reportuseridentity/domain/ReportSummary.java

```diff
@@ -0,0 +1,13 @@
+package uk.gov.di.ipv.core.reportuseridentity.domain;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+import uk.gov.di.ipv.core.library.annotations.ExcludeFromGeneratedCoverageReport;
+
+@Builder
+@ExcludeFromGeneratedCoverageReport
+public record ReportSummary(
+        @JsonProperty("Total P2") long totalP2Identities,
+        @JsonProperty("Total P2 migrated") long totalP2IdentitiesMigrated,
+        @JsonProperty("Total P1") long totalP1Identities,
+        @JsonProperty("Total P0") long totalP0Identities) {}
```

Comment:
Is it easy to include the counts by other info too - i.e. the consitituents?

Perhaps we should test it in build first, and can iterate on the output later

Reply:
Not sure we needed that.  Do u mean total VC's involved to create those total P2 indenties.

Reply:
something like:
```
{
  "dcmaw-passport,address,fraud": 123,
  "dcmaw-drivingPermit,address,fraud": 456,
  "ukPassport,address,fraud,kbv": 789,
   ...
}
```

Reply:
This will be harder if you store it in a record, and not just aggregate in a single scan - we may be able to work that out later.

Reply:
Done req. change to implement this.